quinn-proto/src/config.rs

```diff
@@ -1015,6 +1039,10 @@ impl fmt::Debug for ClientConfig {
         fmt.debug_struct("ClientConfig<T>")
             .field("transport", &self.transport)
             .field("crypto", &"ClientConfig { elided }")
+            .field(
+                "initial_dst_cid_generator",
+                &"ConnectionIdGenerator { elided }",
```

Comment:
`ConnectionIdGenerator` isn't a type, so this is a bit confusing. Maybe just leave it out and switch to `finish_non_exhaustive`?